subprojects/docs/src/snippets/testing/junit-xml-results/kotlin/build.gradle.kts

```
@@ -0,0 +1,34 @@
+plugins {
+    java
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation("junit:junit:4.13")
+}
+
+// tag::configure-location-task[]
+tasks.test {
+    reports {
+        junitXml.outputLocation.set(layout.buildDirectory.dir("test-junit-xml"))
+    }
+}
+// end::configure-location-task[]
+
+// tag::configure-location-convention[]
+project.setProperty("testResultsDirName", "$buildDir/junit-xml")
+// end::configure-location-convention[]
+
+// tag::configure-content[]
+tasks.test {
+    reports.apply {
```

This should work as `AbstractTestTask` has the corresponding `Action` taking method.
```suggestion
    reports {
```